src/core/filters/filterregistry.js

```diff
@@ -1,5 +1,6 @@
 /** @module FilterRegistry */
 
+import { FilterCombinator } from '@yext/answers-core/dist/models/searchservice/request/CombinedFilter';
```

Comment:
Can we add a `TODO (update)` here? This is alright for now, but I want to make sure things are exported in `answers-core` in such a way that people don't have to reach deep into `dist` to get what they need.

Reply:
Yeah totally! I can just use the SDK's built-in `FilterCombinators` for now (they are the same) and we can update all at once?

Reply:
Sure, that works too!